vita3k/gui/src/game_context_menu.cpp

```diff
@@ -164,6 +192,51 @@ void game_context_menu(GuiState &gui, HostState &host, bool *selected, const std
                 ImGui::EndPopup();
             } else
                 ImGui::PopStyleColor();
+
+            // Manual Popup
+            if (manuals_popup)
+                ImGui::OpenPopup("Manual");
+            const auto display_size = ImGui::GetIO().DisplaySize;
+            ImGui::SetNextWindowPos(ImVec2(-5.0f, -5.0f), ImGuiCond_Always);
+            ImGui::SetNextWindowSize(ImVec2(display_size.x + 10.0f, display_size.y + 10.0f), ImGuiCond_Always);
+            if (ImGui::BeginPopupModal("Manual", &manuals_popup, ImGuiWindowFlags_AlwaysAutoResize | ImGuiWindowFlags_NoDecoration | ImGuiWindowFlags_NoMove | ImGuiWindowFlags_NoSavedSettings)) {
+                if (gui.manual.find(manuals_list[current_manual]) == gui.manual.end())
+                    load_manuals(gui, host, manuals_list[current_manual]);
+                else if (gui.current_manual != gui.manual[manuals_list[current_manual]])
+                    gui.current_manual = gui.manual[manuals_list[current_manual]];
+
+                if (gui.current_manual)
+                    ImGui::GetWindowDrawList()->AddImage(reinterpret_cast<void *>(gui.current_manual), ImVec2(0, 0), display_size);
+                else
+                    manuals_popup = false;
+
+                ImGui::SetCursorPos(ImVec2(display_size.x - 40.0f, 20.0f));
+                if (ImGui::Button("X", ImVec2(30.0f, 0.f)))
+                    manuals_popup = false;
+
+                ImGui::SetCursorPos(ImVec2(display_size.x / 2.0f - 90.0f, display_size.y - 40.0f));
```

Comment:
Ditto, constexpr and global is good.

Reply:
no, only for one things so not needed

Reply:
I don't mind if you don't do this, but consider a person looking over your code trying to increase the left padding or something. They can't tell which constants do what.

Like what does 40.0f do here? I'm not sure... If you had all the constants named like `static constexpr float LEFT_PADDING = 40.0f;` then its super easy to tell.

Reply:
hmm i see ^^ but normally you can easy see what is exacly ^^

Reply:
That might be true but names make it easier XD

Reply:
yes, but i keep like that, i don't like create value for call it just one time haha